pkg/cli/printer/json.go

```diff
@@ -19,7 +19,7 @@ func (printer *JSON) Format() string {
 }
 
 func (printer *JSON) Print(w io.Writer, obj any) error {
-	data, err := json.MarshalIndent(obj, "", "    ")
+	data, err := json.MarshalIndent(obj, "", "	")
```

Comment:
Json should be indented with spaces.

Reply:
Alright thanks, I was just about to look it up after I saw your first comment. I'll get that fixed.